### PR TITLE
add and debug login validator to CreateCredentialsCommand

### PR DESCRIPTION
**Pull request by ndoriomedov**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/828

### CODE REVIEW

File: src/UserService.Validation/Login/Interfaces/ILoginValidator.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Attributes;
+
+namespace LT.DigitalOffice.UserService.Validation.Login.Interfaces
+{
+    [AutoInject]
+    public interface ILoginValidator : IValidator<string>
+    {
+
```
Comment (MariyaSin) [inline src/UserService.Validation/Login/Interfaces/ILoginValidator.cs:9]: remove empty line

File: src/UserService.Validation/Login/LoginValidator.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                RuleFor(login => login)
+                .Must(x => char.IsLetter(x[0]))
+                .WithMessage("Login must start with letter")
+                .MinimumLength(3)
+                .WithMessage("Login is too short")
+                .MaximumLength(15)
+                .WithMessage("Login is too long")
+                .Must(x => x.All(char.IsLetterOrDigit))
+                .WithMessage("Login must contain only letters or digits");
+            });
```
Comment (MariyaSin) [inline src/UserService.Validation/Login/LoginValidator.cs:26]: Please, add password and login check

Comment (MariyaSin) [reply src/UserService.Validation/Login/LoginValidator.cs:26]: Password must be notEmpty, userId must notEmpty

File: src/UserService.Validation/Login/Interfaces/ILoginValidator.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.Attributes;
+
+namespace LT.DigitalOffice.UserService.Validation.Login.Interfaces
+{
+    [AutoInject]
+    public interface ILoginValidator : IValidator<string>
```
Comment (BelehovEgor) [inline src/UserService.Validation/Login/Interfaces/ILoginValidator.cs:7]: why validator only for 'login'. make it validator for CreateCredentialsRequest, and validate that UserId exists

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

File: src/UserService.Validation/Login/LoginValidator.cs (hunk 2 above)
Comment (djino97) [reply src/UserService.Validation/Login/LoginValidator.cs:26]: We need validation user login at long?

**Review by djino97 — APPROVED** [review]

File: src/UserService.Validation/Login/Interfaces/ILoginValidator.cs (hunk 1 above)
Comment (ndoriomedov) [reply src/UserService.Validation/Login/Interfaces/ILoginValidator.cs]: done

File: src/UserService.Validation/Login/Interfaces/ILoginValidator.cs (hunk 3 above)
Comment (ndoriomedov) [reply src/UserService.Validation/Login/Interfaces/ILoginValidator.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

**Pull request merged**